administracion/forms.py

```diff
@@ -135,9 +135,14 @@ class FeedbackForm(forms.ModelForm):
     creating this form.
     """
 
+    descripcion = forms.CharField(
```

Comment:
this is not needed since the field "descripcion" in the Meta class is directly binded to the field in the model (i guess you got confused? you can also add the form-control class directly there)